vars/pipelineCapsuleSystemTests.groovy

```diff
@@ -102,8 +102,16 @@ void call() {
               [
                 (name): {
                   childParams = collectParams()
+                  // Collect pytest args, which may be specified in parent, and/or used to collect parallel subjobs
+                  pytestArgs = []
                   if (testSpec.pytestArgs) {
-                    childParams += [string(name: 'TEST_EXTRA_PYTEST_ARGS', value: testSpec.pytestArgs)]
+                    pytestArgs += testSpec.pytestArgs
+                  }
+                  if (params.TEST_EXTRA_PYTEST_ARGS != "") {
```

Comment:
If this if doesn't do anything then let's remove it

Reply:
No, this was me being a moron. I'll fix it. Good spot, thanks.